[EN][FR] Update articles related to game modes and gameplay, add missing links (#12539)
ppy/osu-wiki: wiki/Gameplay/Hit_object/Hit_circle/en.md

# Hit circle

> *Hit circle* are a type of colored, circular [hit object](/wiki/Gameplay/Hit_object) present in [osu!](/wiki/Game_mode/osu!) and [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). They are core gameplay elements, requiring players to click on/tap them in time with the music to [score](/wiki/Gameplay/Score) point. The more [accurate](/wiki/Gameplay/Accuracy) is the player the more points he scores, and it provides a very small boost to the [health bar](/wiki/Gameplay/Health) as well as a larger boost at the end of a [combo](/wiki/Beatmapping/Combo).

Reviewer: we use british english
some of these words are incorrectly put into singular forms <.<

```suggestion
*Hit circles* are a type of coloured, circular [hit objects](/wiki/Gameplay/Hit_object) present in [osu!](/wiki/Game_mode/osu!) and [osu!taiko](/wiki/Game_mode/osu!taiko) [beatmaps](/wiki/Beatmap). They are core gameplay elements, requiring players to click on/tap them in time with the music to [score](/wiki/Gameplay/Score) points. Clicking hit circles more [accurately](/wiki/Gameplay/Accuracy) awards more score, as well as providing a very small boost to the [health bar](/wiki/Gameplay/Health) and a larger one at the end of a [combo](/wiki/Beatmapping/Combo).
```

> ## Customization and skinning
> 
> Hit circles are customizable through osu! [skins](/wiki/Skin), allowing player to modify their appearance. 

Reviewer: again, british english

```suggestion
## Customisation and skinning

Hit circles are customisable through [skins](/wiki/Skin), allowing players to modify their appearance. 
```

## In osu!

In osu!, hit circles are numbered to indicate their order in a combo. Each hit circle is surrounded by an [approach circles](/wiki/Gameplay/Hit_object/Approach_circle), which shrink around it. Once the approach circle overlaps the hit circle, the player must click on/tap the hit cirlce to earn score. Missing this timing results in a lower score based on the timing gap, interrupt the combo and reduces the player's health.

![Hit circles](/wiki/shared/osu_hitcircles.jpg "osu! hit circles")

> ## In osu!taiko
> 
> In osu!taiko, hit circles appear from right-to-left as either red or blue notes (Don or Katu). As hit circles approach and line up with the white [judgement](/wiki/Gameplay/Judgement) circle the correct key(s) must be pressed to gain score.
> 
> ![Taiko hit circles](/wiki/shared/Taiko_hitcircles.jpg "osu!taiko hit circles")
> 

Reviewer: ...osu!taiko doesn't have hit circles. it has different hit objects called "dons" and "kats"

```suggestion
```

Author: I haven't changed the osu taiko part, it was like this before. So have i the right to remove this ? 
If you think so, I will
